docs/index.rst

```diff
@@ -52,8 +52,10 @@ messages, and every 10 seconds.
 
 .. code-block:: python
 
-    import requests
-    from urlparse import urlparse
+    import json
+    import urllib.parse
+    import urllib.request
+    import urlparse
```

Comment:
`urlparse` isn't a module in Python 3, I suspect it would be best to directly import from `urllib.parse`:


```suggestion
    from urllib.parse import urlencode, urlparse
    from urllib.request import urlopen
```

And make the changes below to match. I think that will keep the interior of the task a little simpler.

Reply:
Rebased, updated. How's it now?

Reply:
Looks great! 🎉